Library/Formula/devil.rb

```diff
@@ -1,11 +1,46 @@
 class Devil < Formula
   desc "Cross-platform image library"
   homepage "https://sourceforge.net/projects/openil/"
-  url "https://downloads.sourceforge.net/project/openil/DevIL/1.7.8/DevIL-1.7.8.tar.gz"
-  sha256 "682ffa3fc894686156337b8ce473c954bf3f4fb0f3ecac159c73db632d28a8fd"
   revision 1
 
-  bottle :disable, "Can't generate bottles until builds with either Clang or GCC-5"
+  PATCH_DATA = <<-EOS.undent
```

Comment:
Yup! Merged your PR over there, thanks for submitting those. When you add the links here please reference the commit rather than master, i.e.:

```ruby
patch do
  url "https://raw.githubusercontent.com/Homebrew/patches/3919bbcf3/devil/patch-DevIL-src-ILU-ilur-ilur.c.diff"
  sha256 ""
end
```
And such.